rest-api/core-rest/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,33 +1,71 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<web-app
-        metadata-complete="false"
-        version="3.1">
+<web-app metadata-complete="false" version="3.1">
 
     <display-name>Gaffer REST example</display-name>
 
-
-    <listener>
-        <listener-class>uk.gov.gchq.gaffer.rest.ServletLifecycleListener</listener-class>
-    </listener>
+    <!--<listener>-->
```

Comment:
If this is not needed then can we delete it?

Reply:
I've uncommented these lines as they were added on another branch.